modules/cnf-rfhe-notifications-api-refererence.adoc

```diff
@@ -0,0 +1,321 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+:_content-type: REFERENCE
+[id="cnf-rfhe-notifications-api-refererence_{context}"]
+= Redfish hardware event notifications REST API reference
+
+You can use the Redfish hardware fast event notifications REST API to subscribe an application to Redfish hardware events.
+
+Subscribe distributed unit (DU) applications to Redfish hardware events using the resource address `/cluster/node/<node_name>/redfish/event`, where `<node_name>` is the cluster node where Redfish compliant hardware is located.
+
+The Redfish events notifications REST API is served at `http://localhost:9087/` by default.
+
+[NOTE]
+====
+Only applications on localhost in the pod where the Redfish hardware event occurs can access the hardware events API on port 9087.
+====
+
+To get the name of the `hw-event-proxy` pod where you can access the Redfish hardware events API, perform the following procedure:
+
+* Get the pods running in the namespace where you installed the Hardware Event Proxy Operator, for example, `openshift-hw-events`:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-hw-events
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                         READY   STATUS    RESTARTS       AGE
+hw-event-proxy-99bf99794-jvrxb                               3/3     Running   3              13d
+hw-event-proxy-operator-controller-manager-96fb6f55b-67nqp   2/2     Running   6              13d
+----
+
+The following API endpoints are available:
+
+* `/api/cloudNotifications/v1/publishers`
+- `POST`: Creates a new publisher
+- `GET`: Retrieves a list of publishers
+* `/api/cloudNotifications/v1/publishers/<publisher_id>`
+- `GET`: Creates a new status ping request for the specified publisher ID
+* `/api/cloudNotifications/v1/subscriptions`
+- `POST`: Creates a new subscription
+- `GET`: Retrieves a list of subscriptions
+* `/api/cloudNotifications/v1/subscriptions/<subscription_id>`
+- `GET`: Returns details for the subscription with ID `<subscription_id>`
+* `api/cloudNotifications/v1/subscriptions/status/<subscription_id>`
+- `PUT`: Creates a new status ping request for the specified subscription ID
+* `/api/cloudNotifications/v1/health`
+- `GET`: Returns the health status of the `cloudNotifications` API
+
+[discrete]
+== api/cloudNotifications/v1/publishers
+
+[discrete]
+=== HTTP method
+
+`POST api/cloudNotifications/v1/publishers`
+
+[discrete]
+==== Description
+
+Creates a new publisher. If a publisher is succesfully created, or if it already exists, a `201 Created` status code is returned.
```

Comment:
Customer does not need to call this API.
Add a note:
A publisher is automatically created when a hw-event-proxy CR is created.

Reply:
made these changes - will action your slack suggestions next week